### PR TITLE
Added support for ItemPage

### PR DESCRIPTION
**Pull request by JuanCrg90**
Hi, i added support for the ItemPage parameter.

http://docs.aws.amazon.com/AWSECommerceService/latest/DG/PagingThroughResults.html

And i added a little example of use.

I hope you take into account my change.

Regards.

### CODE REVIEW

**Comment by t3chnoboy** [comment]
Hi!
Looks great! Thank you!
However, there are some points which need to be addressed:
* Delete trailing whitespace
* Remove your example and add itemPage option to any existing example
* Update this section in readme https://github.com/t3chnoboy/amazon-product-api#search-query-options

File: lib/utils.js [hunk 1]
```diff
@@ -12,18 +12,20 @@ exports.generateQueryString = function(query, credentials) {
...
   var domain = query.domain || 'webservices.amazon.com';
   var unsignedString =
     "AWSAccessKeyId=" + credentials.awsId +
     "&AssociateTag=" + credentials.awsTag +
     "&Condition=" + condition +
+    "&ItemPage=" + itemPage +
     "&Keywords=" + (encodeURIComponent(keywords)) +
     "&Operation=ItemSearch&ResponseGroup=" + responseGroup +
-    "&SearchIndex=" + searchIndex +
+    "&SearchIndex=" + searchIndex +
```
Comment (t3chnoboy) [inline lib/utils.js]: trailing whitespace

File: lib/utils.js [hunk 2]
```diff
@@ -12,18 +12,20 @@ exports.generateQueryString = function(query, credentials) {
...
+    "&ItemPage=" + itemPage +
     "&Keywords=" + (encodeURIComponent(keywords)) +
     "&Operation=ItemSearch&ResponseGroup=" + responseGroup +
-    "&SearchIndex=" + searchIndex +
+    "&SearchIndex=" + searchIndex +    
     "&Service=AWSECommerceService&Timestamp=" + (encodeURIComponent((new Date()).toISOString())) +
     "&Version=2011-08-01";
   var signature = encodeURIComponent(generateSignature('GET\n' + domain + '\n/onca/xml\n' + unsignedString, credentials.awsSecret)).replace(/\+/g, '%2B');
-  var queryString = 'http://' + domain + '/onca/xml?' + unsignedString + '&Signature=' + signature;
+  var queryString = 'http://' + domain + '/onca/xml?' + unsignedString + '&Signature=' + signature;
```
Comment (t3chnoboy) [inline lib/utils.js]: trailing whitespace

**Comment by JuanCrg90** [comment]
I made the requested changes. i hope everything is in order.

Greetings.

**Comment by t3chnoboy** [comment]
Awesome! :+1: 
Thank you!

**Comment by t3chnoboy** [comment]
Released as 0.1.5

**Comment by JuanCrg90** [comment]
Thanks to you :smile:

**Pull request merged**